node/core/backing/src/lib.rs

```diff
@@ -915,21 +916,6 @@ impl<Context> CandidateBackingJob<Context> {
 		candidate: &CandidateReceipt,
 		pov: Arc<PoV>,
 	) -> Result<(), Error> {
-		// Check that candidate is collated by the right collator.
-		if self
```

Comment:
@eskimor I don't think we an equivalent check in backing anymore. Is this still needed though?

Reply:
We can drop it.